libbeat/_meta/config.reference.yml

```diff
@@ -778,3 +778,19 @@ logging.files:
 
 # Set to true to log messages in json format.
 #logging.json: false
+
+
+#================================ HTTP Endpoint ======================================
+# Each beat can expose internal data points through a http endpoint. For security
+# reason the endpoint is disabled by default. This feature is currently experimental.
+# Stats can be access through http://localhost:5066/stats For a pretty json output,
+# add ?pretty .
+
+# Defines if http endpoint is enabled
+#http.enabled: false
```

Comment:
Would it make sense to put a more specific name here? `http` is pretty generic, what about `stats.http` or similar?